public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -95,10 +93,10 @@ include ../../../../_includes/_util-fns
   If someone extends the `Engine` class, that is not `Car`'s problem.
 .l-sub-section
   :marked
-    The consumer of `Car` has the problem. The consumer must update the car creation code to
-    something like:
+    The _consumer_ of `Car` has the problem. The consumer must update the car creation code to
+    something like this:
 
-  +makeExample('dependency-injection/ts/app/car/car-creations.ts', 'car-ctor-instantiation-with-param', 'Super Car')(format=".")
+  +makeExample('dependency-injection/ts/app/car/car-creations.ts', 'car-ctor-instantiation-with-param', '')(format=".")
```

Comment:
Without the `Super Car` title, I can't tell that this is a _different_ class - a _derived_ class - with a different implementation.

Why do you think these particular changes are worthwhile?